### PR TITLE
Citation of SIMPA paper

### LINKED ISSUE #124: Add CITATION file
For users who want to cite SIMPA, it would be nice, if there was a citation option directly available in the GitHub repo. Citations can now be easily added to GitHub repos, here is a [guide](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files). It's probably best to add the SIMPA paper as citation object rather than the repo.

### PR DESCRIPTION
**Pull request by TomTomRixRix**
Fixes #124 by adding a citation file and a citation section in the Readme

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    given-names: "Patricia"
+  - family-names: "Ayala"
+    given-names: "Leonardo"
+  - family-names: "Bohndiek"
+    given-names: "Sarah E."
+  - family-names: "Seitel"
+    given-names: "Alexander"
+  - family-names: "Maier-Hein"
+    given-names: "Lena"
+  doi: "0.1117/1.JBO.27.8.083010"
```
Comment (Holzwarth69126) [inline CITATION.cff:49]: DOI wrong - add a 1

File: README.md [hunk 2]
```diff
@@ -208,4 +209,22 @@ If you encounter an error similar to:
...
 
 Look up the solution in [this thread of the k-Wave forum](http://www.k-wave.org/forum/topic/error-reading-h5-files-when-using-binaries).  
-      
\ No newline at end of file
+      
+# Citation
+
+If you use the SIMPA tool, we would appreciate if you cite our Journal publication in the Journal of Biomedical Optics:
+
+Gröhl, Janek, Kris K. Dreher, Melanie Schellenberg, Tom Rix, Niklas Holzwarth, Patricia Vieten, Leonardo Ayala, Sarah E. Bohndiek, Alexander Seitel, and Lena Maier-Hein. *"SIMPA: an open-source toolkit for simulation and image processing for photonics and acoustics."* **Journal of Biomedical Optics** 27, no. 8 (2022): 083010.
```
Comment (Holzwarth69126) [inline README.md:217]: Delete 083010?

**Review by Holzwarth69126 — CHANGES_REQUESTED** [review]
Please update

**Review by Holzwarth69126 — APPROVED** [review]
Works 🥝

**Review by kdreher — APPROVED** [review]
looks good to me

**Pull request merged**